docs/proposals/202005_version_documentation.md

```diff
@@ -86,11 +86,11 @@ We want to add an additional `versioned` folder within the website's `tmp` direc
     |- tmp
         |- public
         |- docs-pre-processed
-        |- versioned
-            |- master
-            |- version 0.13.0
-            |- version 0.12.2
-            |- other-folder-for-other-releases
+            |- versioned
```

Comment:
That's not true, right? (:

Reply:
True...it's just that I think it's best we identify where those versioned folders are located (which in this case, exists in docs-pre-processed)

Anyways...those details don't matter much